app/views/users/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<h4><%= @user.username %></h4>
+
+<div>
+  <img src="<%= @user.avatar %>">
+</div>
+<div data-role="follow">
+  <button data-role="new-follow" value="<%= @user %>">follow</button>
```

Comment:
I am wondering why you have a data-role on both the container and the button itself. Do you have a click button on the button and then you want to hid the div after a click? You might not need a container for this? I duno :confounded: